examples/denoise_covar2d.py

```diff
@@ -168,4 +169,17 @@
 logger.info(f'Deviation of the noisy covariance estimate: {diff_covar}')
 logger.info(f'Estimated images normalized RMSE: {nrmse_ims}')
 
-
+# plot the first images at different stages
+plt.subplot(2, 2, 1)
+plt.imshow(np.real(imgs_noise[..., 0]), cmap='gray')
+plt.title('Noise')
+plt.subplot(2, 2, 2)
+plt.imshow(np.real(-imgs_clean[..., 0]), cmap='gray')
```

Comment:
It looks that the simulated noisy images are not phase flipped. The minus sign is just used to make the consistent representation of images.

Reply:
Ok. If there's a sign problem, it should be fixed in the simulation. It just looks strange here because the noisy images are not flipped, only the clean ones.

Reply:
Let's make an issue.

Reply:
Opened an new issue #91 to fix it later. Closed it now.